packages/eslint-config-ckeditor5/.eslintrc.js

```diff
@@ -308,6 +308,12 @@ module.exports = {
 		'yield-star-spacing': [
 			'error',
 			'after'
+		],
+		'ckeditor5-rules/require-file-extensions-in-imports': [
+			'error',
+			{
+				extensions: [ '.ts', '.js', '.json' ]
+			}
 		]
```

Comment:
I think, this rule should be enabled directly in the CKEditor 5 projects. This package/config is shared in all our repositories, but only some of them will be exported as ESM.